python/perspective/perspective/handlers/aiohttp.py

```diff
@@ -48,10 +48,14 @@ async def run(self) -> None:
             self.on_close()
 
     async def write_message(self, message: str, binary: bool = False) -> None:
-        if binary:
-            await self._ws.send_bytes(message)
-        else:
-            await self._ws.send_str(message)
+        try:
+            if binary:
+                await self._ws.send_bytes(message)
+            else:
+                await self._ws.send_str(message)
+        except WebSocketError:
```

Comment:
I don't know enough about the ecosystem of possible exceptions here, but I have spent countless hours of my life chasing bugs that would've been immediately obvious had they not been swallowed by a blanket catch block. At minimum, some comments about what to expect in these cases and why would be super helpful - but I propose we generally try to handle specific exceptions instead, which would also communicate intent here.

Reply:
Maybe also in this case merits a `debug` log, now that I think about it.